docs/azure/sdk/dependency-injection.md

```diff
@@ -106,43 +116,97 @@ In the [Register client](#register-client) section, you explicitly specify the `
 }
 ```
 
-You can add any options from <xref:Azure.Core.ClientOptions> into the JSON file's `AzureDefaults` section. One of the options is the retry policy. For more information, see [Configure a new retry policy](#configure-a-new-retry-policy).
+You can add any properties from the <xref:Azure.Core.ClientOptions> class into the JSON file's `AzureDefaults` section. In the preceding JSON sample, the `AzureDefaults.Retry` object literal:
+
+* Represents the retry policy configuration settings.
+* Corresponds to the <xref:Azure.Core.ClientOptions.Retry> property. Within that object literal, you find the `MaxRetries` key, which corresponds to the <xref:Azure.Core.RetryOptions.MaxRetries> property.
 
-Since the `Configuration` object is a member of the `builder` instance, you can configure secrets:
+For more information, see [Configure a new retry policy](#configure-a-new-retry-policy).
+
+The settings in the JSON configuration file can be retrieved using <xref:Microsoft.Extensions.Configuration.IConfiguration>.
+
+### [WebApplicationBuilder](#tab/web-app-builder)
 
 ```csharp
 builder.Services.AddAzureClients(clientBuilder =>
 {
-    // Add a KeyVault client
-    clientBuilder.AddSecretClient(builder.Configuration.GetSection("KeyVault"));
+    clientBuilder.AddSecretClient(
+        builder.Configuration.GetSection("KeyVault"));
 
-    // Add a storage account client
-    clientBuilder.AddBlobServiceClient(builder.Configuration.GetSection("Storage"));
+    clientBuilder.AddBlobServiceClient(
+        builder.Configuration.GetSection("Storage"));
 
-    // Use DefaultAzureCredential by default
     clientBuilder.UseCredential(new DefaultAzureCredential());
 
     // Set up any default settings
-    clientBuilder.ConfigureDefaults(builder.Configuration.GetSection("AzureDefaults"));
+    clientBuilder.ConfigureDefaults(
+        builder.Configuration.GetSection("AzureDefaults"));
 });
+```
 
-builder.Services.AddControllers();
+### [HostApplicationBuilder](#tab/host-app-builder)
+
+```csharp
+builder.Services.AddAzureClients(clientBuilder =>
+{
+    clientBuilder.AddSecretClient(
+        builder.Configuration.GetSection("KeyVault"));
+
+    clientBuilder.AddBlobServiceClient(
+        builder.Configuration.GetSection("Storage"));
+
+    clientBuilder.UseCredential(new DefaultAzureCredential());
+
+    // Set up any default settings
+    clientBuilder.ConfigureDefaults(
+        builder.Configuration.GetSection("AzureDefaults"));
+});
 ```
```

Comment:
Do we need to mention anything here about whether GetSection relies on naming conventions for properties within the section, such as "ServiceUri" or "VaultUri", since we mentioning naming about the ClientOptions.

Reply:
Great call out. I'll add something.